src/Admin/AdminHelper.php

```diff
@@ -200,7 +205,9 @@ public function appendFormFieldElement(AdminInterface $admin, $subject, $element
         if (\count($toDelete) > 0) {
             $i = 0;
             foreach ($finalForm->get($childFormBuilder->getName()) as $childField) {
-                $childField->get('_delete')->setData(isset($toDelete[$i]) && $toDelete[$i]);
+                if ($childField->has(self::FORM_FIELD_DELETE)) {
+                    $childField->get(self::FORM_FIELD_DELETE)->setData(isset($toDelete[$i]) && $toDelete[$i]);
```

Comment:
Isn't that the same as

```suggestion
                    $childField->get(self::FORM_FIELD_DELETE)->setData($toDelete[$i] ?? false);
```

?